e2e/k8s_test.go

```diff
@@ -29,3 +100,48 @@ func TestSecretsProvidedK8s(t *testing.T) {
 
 	testenv.Test(t, f.Feature())
 }
+
+func TestVariablesWithBase64DecodingSecretProvidedK8s(t *testing.T) {
+	f := features.New("variables with base64 decoding secret provided").
+		// Replaces TEST_ID_1.6_providing_variables_with_base64_decoding_successfully
+		Assess("variables with base64 decoding secret set correctly in pod", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			var stdout, stderr bytes.Buffer
+			command := []string{"printenv", "|", "grep", "VARIABLE_WITH_BASE64_SECRET"}
+
+			RunCommandInSecretsProviderPod(cfg.Client(), command, &stdout, &stderr)
+
+			assert.Contains(t, stdout.String(), "secret-value")
+
+			return ctx
+		})
+
+	testenv.Test(t, f.Feature())
+}
+
+func TestLargeDecodedVariablesSecretProvidedK8s(t *testing.T) {
+	f := features.New("large decoded variables secret provided").
+		// Replaces TEST_ID_1.7_providing_large_decoded_variable_successfully
+		Assess("large decoded variable secret set correctly in pod", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			// set encoded value in conjur
+			value := "secret-value"
+			encoded := base64.StdEncoding.EncodeToString([]byte(val))
+
+			err := SetConjurSecret(cfg.Client(), "VARIABLE_WITH_BASE64_SECRET", encoded)
```

Comment:
Since this is running in a mode without secrets rotation, updating a secret value will not be reflected in the container unless you redeploy the secrets provider pod. It just so happens that the `VARIABLE_WITH_BASE64_SECRET` is already equal to `secret-value` from the original deployment. 

It is also intended to test decoding a value larger than 65k characters.

The flow for this test should be:

1. Generate a random string > 65k characters
2. Base64 encode the string
3. Set Conjur secret `secrets/encoded` to your encoded string
4. Re-deploy Secrets Provider pod (`ReloadWithTemplate(cfg.Client(), K8sTemplate)` should do it, I think)
5. Check the environment variable for expected value